makiflow/layers/untrainable_layers.py

```diff
@@ -698,20 +756,22 @@ def __init__(self, name, p_keep=0.9, noise_shape=None, seed=None):
         self._p_keep = p_keep
         self.noise_shape = noise_shape
         self.seed = seed
+
         super().__init__(name, params=[],
                          regularize_params=[],
                          named_params_dict={}
         )
 
-
-    def _forward(self, x):
-        return x
+    def _forward(self, X, type_graph_operation=MakiRestorable.TEST_PREFIX):
+        with tf.name_scope(type_graph_operation + super().get_name()):
```

Comment:
This scope won't affect anything. It is needed to use `tf.identity(X)` in this case.